postcss.config.js

```diff
@@ -4,6 +4,6 @@
 module.exports = {
     plugins: [
         require('autoprefixer'),
-        require('tailwindcss')('./tailwind.config.js')
+        require('tailwindcss')('../venia-concept/tailwind.config.js')
```

Comment:
@jcalcaben Just curious how this works. `../` would put it outside of the project no?

Reply:
I think it's a quirk of the monorepo setup. When we build or run storybook it uses the `postcss.config.js` at the root of the project, but the working directory is within that package (either `venia-concept` for development or `venia-ui` for storybook). At least that's my best guess to how it works.

Reply:
Got it. Thanks! 😄